src/replacer.coffee

```diff
@@ -55,7 +58,8 @@ class Replacer
   _replaceArray: (orig) ->
     arr = (@replace(item) for item in orig)
     # Convert the nextPage methods for paged results
-    for key, value of orig
+    for key in Object.keys(orig)
+      value = orig[key]
       @_replaceKeyValue(arr, key, value) if typeof key is 'string'
```

Comment:
Thanks for catching the type check!

`_replaceKeyValue` recursively goes through the values and converts them. So, in the case of  the value being an array or object, it will continue converting them (for example the `owner` key/value when retrieving a repository).